src/main/java/reposense/RepoSense.java

```diff
@@ -92,6 +92,8 @@ public static void main(String[] args) {
             }
         }
 
+        RepoConfiguration.setStandaloneConfigIgnoredToRepoConfigs(configs, cliArguments.isStandaloneConfigIgnored());
```

Comment:
This should probably be set in the main method, so that either `LocationCliArguments` or `ConfigCliArguments` will set this boolean value (override the values in the config file, since they are `false` by default as well).

Reply:
Refer to #390, I am not sure with the level of argument overriding

Reply:
Hmmm okay, can ignore this comment for now.